engine/src/main/java/org/terasology/world/internal/WorldProviderCore.java

```diff
@@ -163,9 +164,10 @@
      * @return The (index)th extra-data value at the given position
      */
     int getExtraData(int index, int x, int y, int z);
-    
+
     /**
      * Sets one of the per-block custom data values at the given position, if it is within the view.
+     * Do not use this method with world gen code, pass the chunk as an argument instead.
```

Comment:
```suggestion
     * You must not use this method with world gen code, call `setExtraData` on chunk instead.
```

Reply:
Just, you know, use whatever the javadoc syntax for linking methods is, instead of the quotes